lib/playground.dart

```diff
@@ -537,6 +541,10 @@ class Playground implements GistContainer, GistController {
   /// Perform static analysis of the source code. Return whether the code
   /// analyzed cleanly (had no errors or warnings).
   Future<bool> _performAnalysis() {
+
+    // We only have issues for dart source code.
+    if (context.focusedEditor != "dart") return null;
```

Comment:
And if it is necessary, we should change to returning `new Future.value(true)` (to tell clients the the code analyzed cleanly).